components/calculator.jsx

```diff
@@ -0,0 +1,53 @@
+// import React from 'react'
+import React, { useState } from 'react'
+
+export default () => {
+  const [num1, setNum1] = useState('')
+  const [num2, setNum2] = useState('')
+  const [operator, setOperator] = useState('')
+  const [answer, setAnswer] = useState('')
+  const [error, setError] = useState('')
+
+  const calculateAnswer = () => {
+    if (Number(num1) && Number(num1)) {
+      if (error) setError('')
+
+      switch (operator) {
+        case '+':
+          setAnswer(Number(num1) + Number(num2))
+          break
+        case '-':
+          setAnswer(Number(num1) - Number(num2))
+          break
+        case '*':
+          setAnswer(Number(num1) * Number(num2))
+          break
+        case '/':
+          setAnswer(Number(num1) / Number(num2))
+          break
+        default:
+      }
+    } else {
+      setError('Enter a numeric in both boxes')
+    }
+  }
+
+  return (
+    <div className="page">
+      <div className="title">React Calculator</div>
+      <className input="num1" type="text" name="num1" onChange={event => setNum1(event.target.value)} />
```

Comment:
_Nit_ youre missing the input tag here so youre missing a field on your calculator.